docs/iceberg-rest-service.md

```diff
@@ -420,6 +421,18 @@ Gravitino provides a pluggable metrics store interface to store and delete Icebe
 | `gravitino.iceberg-rest.metricsStoreRetainDays` | The days to retain Iceberg metrics in store, the value not greater than 0 means retain forever.                                     | -1            | No       | 0.4.0         |
 | `gravitino.iceberg-rest.metricsQueueCapacity`   | The size of queue to store metrics temporally before storing to the persistent storage. Metrics will be dropped when queue is full. | 1000          | No       | 0.4.0         |
 
+### Iceberg table metadata cache configuration
+
+Gravitino features a pluggable cache system for updating or retrieving table metadata caches. It validates the location of table metadata against the catalog backend to ensure the correctness of cached data.
+
+| Configuration item                                           | Description                                 | Default value | Required | Since Version |
+|--------------------------------------------------------------|---------------------------------------------|---------------|----------|---------------|
+| `gravitino.iceberg-rest.table-metadata-cache-impl`           | The implement of the cache.                 | (none)        | No       | 1.1.0         |
+| `gravitino.iceberg-rest.table-metadata-cache-capacity`       | The capacity of table metadata cache.       | 200           | No       | 1.1.0         |
+| `gravitino.iceberg-rest.table-metadata-cache-expire-minutes` | The expire minutes of table metadata cache. | 60            | No       | 1.1.0         |
+
+Gravitino provides build-in `org.apache.gravitino.iceberg.common.cache.LocalMetadataCache` to store the cache data in local memory. And you could implement your custom cache by implementing `org.apache.gravitino.iceberg.common.cache.MetadataCache` interface.
```

Comment:
"provides the built-in ... to store the cached data in the memory. You could also implement your custom table metadata cache by implementing the ..."

Reply:
updated